config/vta_cost.py

```diff
@@ -0,0 +1,102 @@
+# cost function for intelfocl 32*32 gemm version
+def cal_cost(insn):
+    """
+    Cal the runtime cost statically
+
+    Parameters
+    ------------
+    insn: the insn (json)
+
+    Returns
+    ------------
+    the cost in s
+    """
+    factor = 1000000.0
+    def alu_imm_cost(iter_out, iter_in, uop_bgn, uop_end):
+        x = (uop_end - uop_bgn) * iter_out * iter_in
+        cycles = x + 46
```

Comment:
How are these costs generally derived? Are they FPGA device specific? If we change the compilation parameters in the OpenCL compiler, could it affect these cost models?

Reply:
Yes. Those costs are FPGA and configuration specific, and we need to profile/measure the values from the actual hardware.

I believe the values are provided here as an example.

Reply:
Actually I think it would be good to define constants here, and add a comment that states that these are set to arbitrarily (e.g. 46).

I'm working with a colleague on making easier to perform some of these cycle accurate co-simulation of VTA modules which should help us derive these costs more accurately on a per-VTA variant basis. In the meantime, can we use constant names, e.g. `ALU_LATENCY_CYCLES = 46`